inbm/dispatcher-agent/dispatcher/schedule/schedules.py

```diff
@@ -9,17 +9,22 @@
 from typing import Optional
 
 @dataclass(init=True)
-class SingleSchedule:
-    """ Represents a SingleSchedule object from the Scheduled Manifest """
+class Schedule:
+    """ Represents a Base class for schedule objects """
     request_id: str
+    id: Optional[int] = field(default=None)    
+
+
+@dataclass(init=True)
+class SingleSchedule(Schedule):
+    """ Represents a SingleSchedule object from the Scheduled Manifest """ 
     start_time: Optional[datetime] = field(default=None)
     end_time: Optional[datetime] = field(default=None)
     manifests: list[str] = field(default_factory=list)
  
 @dataclass(init=True)
-class RepeatedSchedule:
+class RepeatedSchedule(Schedule):
     """ Represents a RepeatedSchedule object from the Scheduled Manifest """
-    request_id: str
     cron_duration: str = field(default='*')
```

Comment:
**<h2>$\color{#FFD700}Coding Style$</h2>**
**Suggestion:** Utilize a more descriptive default value for cron_duration and other cron fields to enhance code readability and intent.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    cron_duration: str = field(default='* * * * * *')
```